shell/platform/darwin/macos/framework/Source/FlutterAppDelegate.mm

```diff
@@ -18,6 +24,13 @@ @implementation FlutterAppDelegate
 // TODO(stuartmorgan): Implement application lifecycle forwarding to plugins here, as is done
 // on iOS. Currently macOS plugins don't have access to lifecycle messages.
```

Comment:
Consider updating the TODO owner and linking to https://github.com/flutter/flutter/issues/30735

Reply:
Okay, done.  This TODO will hopefully not last long, since my next task for the lifecycle implementation is to do exactly that.